recruitment/models.py

```diff
@@ -101,10 +101,8 @@ def handle_answers_from_request(self, request, user):
                         user=user
                     )
                     answer_string = request.POST[key]
-
-                    if answer_string:
-                        answer.answer = answer_string
-                        answer.save()
+                    answer.answer = answer_string
+                    answer.save()
```

Comment:
I still think it's better to err on the side of caution, but I get what you mean :)